src/main/groovy/info/solidsoft/gradle/pitest/PitestTask.groovy

```diff
@@ -249,10 +249,14 @@ class PitestTask extends JavaExec {
     @Optional
     List<String> overriddenTargetTests  //should be Set<String> or SetProperty but it's not supported in Gradle as of 5.6.1
 
+    @Internal
+    File rootDir
+
     PitestTask() {
         getMainClass().set("org.pitest.mutationtest.commandline.MutationCoverageReport")
 
         ObjectFactory of = project.objects
+        rootDir = project.rootDir
```

Comment:
It mimic the original logic, so let keep it as is in that PR. However, I feel in by bones it might be needed to make it configurable one day (and set in the `PitestPlugin` class, not in the `PitestTest` constructor).